### PR TITLE
Feat/add new prop in inputnumber

### PR DESCRIPTION
**Pull request by planivic1**
### Description
in this PR, I add a new prop in number input to don't show virtual keyboard

### CODE REVIEW

**Review by ixpd — APPROVED** [review]
LGTM

File: src/components/Inputs/NumberInput/NumberInput.stories.mdx [hunk 1]
```diff
@@ -22,6 +22,10 @@ import { colors } from '@pb/utils/constants.js';
       type: 'boolean',
       defaultValue: false,
     },
+    disableTouchKeyboard: {
```
Comment (mickaelnn) [inline src/components/Inputs/NumberInput/NumberInput.stories.mdx:25]:
```suggestion
    disableSoftKeyboard: {
```

File: src/components/Inputs/NumberInput/NumberInput.stories.mdx [hunk 2]
```diff
@@ -72,6 +76,7 @@ export const Template = (args, { argTypes}) => ({
         v-model="value"
         :placeholder="placeholder"
         :disabled="disabled"
+        :disableTouchKeyboard="disableTouchKeyboard"
```
Comment (mickaelnn) [inline src/components/Inputs/NumberInput/NumberInput.stories.mdx:79]:
```suggestion
        :disableSoftKeyboard="disableSoftKeyboard"
```

File: src/components/Inputs/NumberInput/NumberInput.vue [hunk 3]
```diff
@@ -31,6 +32,8 @@ export default {
 
     disabled: { type: Boolean, default: false },
 
+    disableTouchKeyboard: { type: Boolean, default: false },
```
Comment (mickaelnn) [inline src/components/Inputs/NumberInput/NumberInput.vue:35]:
```suggestion
    disableSoftKeyboard: { type: Boolean, default: false },
```

File: src/components/Inputs/NumberInput/NumberInput.vue [hunk 4]
```diff
@@ -61,6 +64,16 @@ export default {
         this.setValue(newVal);
       },
     },
+
+    onFocusBlur: {
+      get() {
+        return this.disableTouchKeyboard ? 'blur()' : true;
```
Comment (mickaelnn) [inline src/components/Inputs/NumberInput/NumberInput.vue:70]:
```suggestion
        return this.disableSoftKeyboard ? 'blur()' : true;
```

**Review by mickaelnn — CHANGES_REQUESTED** [review]

File: src/components/Inputs/NumberInput/NumberInput.vue (hunk 4 above)
Comment (mickaelnn) [reply src/components/Inputs/NumberInput/NumberInput.vue:70]: Coloca essa validação numa computed, e retorna a computed: Dica de mestre

**Review by mickaelnn — CHANGES_REQUESTED** [review]

**Review by ixpd — APPROVED** [review]

File: src/components/Inputs/NumberInput/NumberInput.vue [hunk 5]
```diff
@@ -57,6 +60,19 @@ export default {
         return this.value;
       },
 
+      set(newVal) {
+        this.setValue(newVal);
+      },
+    },
+    focusBlur() {
+      return this.disableTouchKeyboard ? 'blur()' : true;
```
Comment (mickaelnn) [inline src/components/Inputs/NumberInput/NumberInput.vue:68]:
```suggestion
      return this.disableSoftKeyboard ? 'blur()' : true;
```

**Review by mickaelnn — CHANGES_REQUESTED** [review]

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by ixpd — APPROVED** [review]

**Pull request merged**